modules/drivers/mongo/src/metabase/driver/mongo/query_processor.clj

```diff
@@ -404,11 +405,11 @@
 
 (defmethod ->lvalue :coalesce [[_ & args]] (->lvalue (first args)))
 
-(defmethod ->rvalue :avg       [[_ inp]] {"$avg" (->rvalue inp)})
-(defmethod ->rvalue :stddev    [[_ inp]] {"$stdDevPop" (->rvalue inp)})
+(defmethod ->rvalue :avg       [[_ inp]] {$avg (->rvalue inp)})
+(defmethod ->rvalue :stddev    [[_ inp]] {"$stdDevSamp" (->rvalue inp)})
```

Comment:
I don't know why this was already in here. But I changed it to use sample sample deviation rather than population standard deviation, because that's what we use for all of our other drivers